discovery-utils/proto/discovery.proto

```diff
@@ -29,6 +32,18 @@ message RegisterDiscoveryHandlerRequest {
 message Empty {
 }
 
+message QueryDeviceInfoRequest{
+    //the basic device information that discovery handler can get is used in this request, such as device ipaddress, mac address etc. It will be a JSON string and it will be directly send to external device inventory system REST API. It is external device inventory system's resonsibility to understand this query_device_payload JSON string and respond.
+    string query_device_payload = 1; 
```

Comment:
can these names be simplified to remove redundant `query_device`?

Reply:
Ok.